src/Main.java

```diff
@@ -0,0 +1,166 @@
+import java.util.Random;
+import java.util.Scanner;
+
+public class Main {
+
```

Comment:
General recommendations:
1. Always format your code
2. Do not leave empty lines and unnecessary spaces (for example `public void stringReverse (String simpleString))`
3. .gitignore must be added and pushed first before any other files! Files and folders, like .idea/, *.iml, out/ must be deleted from this repository.
4. Do not use System.out.println inside your methods! It's better when your method has a return value. print statements can be used for testing only in the main method.